client/src/components/Header.js

```diff
@@ -22,8 +22,7 @@ class Header extends React.Component{
           {
             localStorage.getItem('token') ? (
               <div>
-                <li>Hello {this.props.auth.githubUserId}!</li>
-                <li><button onClick={() => {this.props.logout(); this.props.history.push('/')}}>Logout</button></li>
+                <h6>Hello {this.props.auth.githubUserId}! <button onClick={() => {this.props.logout(); this.props.history.push('/')}}>Logout</button></h6>
```

Comment:
consider to put those to a named function and then call it here, as the methods are growing.